include/interface/blas2_interface.h

```diff
@@ -190,6 +190,32 @@ typename sb_handle_t::event_t _syr(
     index_t _lda             // >max(1, _N) The first dimension of _mA
 );
 
+/*!
```

Comment:
Thanks for your patch, @muhammad-tanvir-1211!

Probably, it would be better to use the doxygen format for documenting the function arguments, as discussed [here](https://github.com/codeplaysoftware/sycl-blas/pull/362#discussion_r1049516404).